src/Nfield.Quota/Helpers/CollectionExtensions.cs

```diff
@@ -23,5 +23,17 @@ public static void AddRange<T>(this ICollection<T> collection, IEnumerable<T> it
                 collection.Add(item);
             }
         }
+
+        /// <summary>
+        /// Compares two definitions' collection if they have the same items and count
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="left"></param>
+        /// <param name="right"></param>
+        /// <returns></returns>
+        public static bool ScrambledDefinitionsEquals<T>(this ICollection<T> left, ICollection<T> right)
```

Comment:
IMO this class should be internal, but since it isn't we can at least make this extension method internal. Kind of strange that we're polluting other people's apps with these. We should really care more about that when writing library code for other people.